packages/ember-htmlbars/tests/helpers/render_test.js

```diff
@@ -1,603 +0,0 @@
-    });
-  }, /Please refactor [\w\{\}"` ]+ to a component/);
-
-  let postController;
-  let PostController = EmberController.extend({
-    init() {
-      this._super(...arguments);
-      postController = this;
-    }
-  });
-  appInstance.register('controller:post', PostController);
-
-  setTemplate('post', compile('<p>{{title}}</p>'));
-
-  runAppend(view);
-
-  runDestroy(view);
-
-  ok(postController.isDestroyed, 'expected postController to be destroyed');
-});
-
-QUnit.test('{{render}} helper should not treat invocations with falsy contexts as context-less', function() {
-  let template = '<h1>HI</h1> {{render \'post\' zero}} {{render \'post\' nonexistent}}';
-  let component;
-
-  expectDeprecation(() => {
-    component = Component.create({
-      [OWNER]: appInstance,
-      zero: false,
-      layout: compile(template)
-    });
-  }, /Please refactor [\w\{\}"` ]+ to a component/);
-
-  let postController1, postController2;
-  let PostController = EmberController.extend({
-    init() {
-      this._super(...arguments);
-      if (!postController1) {
-        postController1 = this;
-      } else if (!postController2) {
-        postController2 = this;
-      }
-    }
-  });
-  appInstance.register('controller:post', PostController, { singleton: false });
-
-  setTemplate('post', compile('<p>{{#unless model}}NOTHING{{/unless}}</p>'));
-
-  runAppend(component);
-
-  ok(component.$().text().match(/^HI ?NOTHING ?NOTHING$/));
-  equal(postController1.get('model'), 0);
-  equal(postController2.get('model'), undefined);
-});
-
-QUnit.test('{{render}} helper should render templates both with and without models', function() {
-  let template = '<h1>HI</h1> {{render \'post\'}} {{render \'post\' post}}';
-  let post = {
-    title: 'Rails is omakase'
-  };
-  let component;
-
-  expectDeprecation(() => {
-    component = Component.create({
-      [OWNER]: appInstance,
-      post,
-      template: compile(template)
-    });
-  }, /Please refactor [\w\{\}"` ]+ to a component/);
-
-  let postController1, postController2;
-  let PostController = EmberController.extend({
-    init() {
-      this._super(...arguments);
-      if (!postController1) {
-        postController1 = this;
-      } else if (!postController2) {
-        postController2 = this;
-      }
-    }
-  });
-  appInstance.register('controller:post', PostController, { singleton: false });
-
-  setTemplate('post', compile('<p>Title:{{model.title}}</p>'));
-
-  runAppend(component);
-
-  ok(component.$().text().match(/^HI ?Title: ?Title:Rails is omakase$/));
-  equal(postController1.get('model'), null);
-  equal(postController2.get('model'), post);
-
-  runSet(component, 'post', { title: 'Rails is unagi' });
-
-  ok(component.$().text().match(/^HI ?Title: ?Title:Rails is unagi$/));
-  deepEqual(postController2.get('model'), { title: 'Rails is unagi' });
-});
-
-QUnit.test('{{render}} helper should be able to render a template again when it was removed', function() {
```

Comment:
Removed. This seems very weird, but maybe this just needs to be an `ApplicationTest`?